libevmcore/EVMSchedule.h

```diff
@@ -63,11 +63,34 @@ struct EVMSchedule
 	unsigned txDataZeroGas = 4;
 	unsigned txDataNonZeroGas = 68;
 	unsigned copyGas = 3;
+
+	//EIP150
+	unsigned extcodesizeGas = 20;
+	unsigned extcodecopyGas = 20;
+	unsigned balanceGas = 20;
+	unsigned suicideGas = 0;
+
+	bool isEIP150() const { return callGas == 700; }
```

Comment:
Previously, we had booleans to trigger certain behaviour, so I think it would be better to replace this by
`bool staticCallDepthLimit = true`, set the value to `false` for the `EVMScheduleEIP150` and replace all current calls to `isEIP150()` by `!staticCallDepthLimit`.